common/types_test.go

```diff
@@ -346,21 +346,21 @@ func TestAddress_Value(t *testing.T) {
 		0xa2, 0x18, 0xc6, 0xa9, 0x27, 0x4d, 0x30, 0xab, 0x9a, 0x15,
 	}
 	var usedA Address
-	usedA.inner.setBytes(b)
+	usedA = BytesToAddress(b);
```

Comment:
I think you dropped an extra semicolon here

Reply:
removed